### PR TITLE
Powsybl 3.7.1

### PR DESCRIPTION
**Pull request by murgeyseb**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java [hunk 1]
```diff
@@ -6,61 +6,26 @@
...
-    static SystematicSensitivityResult runSensitivity(Network network,
-                                                               String workingStateId,
-                                                               CnecSensitivityProvider cnecSensitivityProvider,
-                                                               SensitivityComputationParameters sensitivityComputationParameters) {
-        return runSensitivity(network, workingStateId, cnecSensitivityProvider, sensitivityComputationParameters, DefaultComputationManagerConfig.load().createLongTimeExecutionComputationManager());
-    }
-
-    private static boolean initialised() {
-        return sensitivityComputationFactory != null && computationManager != null;
+        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, workingStateId, cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
```
Comment (TheMaskedTurtle) [inline sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java:26]: Can we even remove workingStateId ? As we always use network working variant

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java [hunk 2]
```diff
@@ -6,61 +6,26 @@
...
 import org.slf4j.LoggerFactory;
 
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
-
 /**
  * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
  */
 final class SystematicSensitivityService {
```
Comment (TheMaskedTurtle) [inline sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java:18]: Maybe we can rename into SystematicSensitivityAdapter :) Even merge it into SystematicSensitivityInterface as it is really small now

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java [hunk 3]
```diff
@@ -6,61 +6,26 @@
  */
 package com.farao_community.farao.sensitivity_computation;
```
Comment (TheMaskedTurtle) [inline sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java:7]: Rename package ?

File: loopflow-computation/pom.xml [hunk 4]
```diff
@@ -41,7 +41,11 @@
         </dependency>
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-sensitivity-api</artifactId>
+            <artifactId>powsybl-sensitivity-analysis-api</artifactId>
```
Comment (TheMaskedTurtle) [inline loopflow-computation/pom.xml:44]: I think there is no need of this module here

Comment (TheMaskedTurtle) [reply loopflow-computation/pom.xml:44]: It doesn't appear here but in this pom I think you can also remove iidm-impl and loadflow-api

File: sensitivity-computation/pom.xml [hunk 5]
```diff
@@ -31,9 +31,17 @@
             <artifactId>farao-util</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
+            <artifactId>auto-service</artifactId>
+        </dependency>
```
Comment (TheMaskedTurtle) [inline sensitivity-computation/pom.xml:37]: Is that useful ?

File: data/glsk/glsk-file/pom.xml [hunk 6]
```diff
@@ -43,7 +43,7 @@
         </dependency>
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-sensitivity-api</artifactId>
+            <artifactId>powsybl-sensitivity-analysis-api</artifactId>
```
Comment (TheMaskedTurtle) [inline data/glsk/glsk-file/pom.xml:46]: glsk file should not rely on computation module no ?

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: loopflow-computation/pom.xml (hunk 4 above)
Comment (murgeyseb) [reply loopflow-computation/pom.xml:44]: Needed for parameters ...

File: sensitivity-computation/pom.xml (hunk 5 above)
Comment (murgeyseb) [reply sensitivity-computation/pom.xml:37]: Yes but in scope test

File: data/glsk/glsk-file/pom.xml (hunk 6 above)
Comment (murgeyseb) [reply data/glsk/glsk-file/pom.xml:46]: It allows to generate LinearGlsk variable... so it is needed

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java (hunk 1 above)
Comment (murgeyseb) [reply sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java]: done

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java (hunk 2 above)
Comment (murgeyseb) [reply sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java]: done

File: sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java (hunk 3 above)
Comment (murgeyseb) [reply sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityService.java]: done

File: sensitivity-computation/pom.xml (hunk 5 above)
Comment (murgeyseb) [reply sensitivity-computation/pom.xml]: done

File: sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityAdapter.java [hunk 7]
```diff
@@ -15,19 +15,18 @@
...
 
     static SystematicSensitivityResult runSensitivity(Network network,
-                                                               String workingStateId,
                                                                CnecSensitivityProvider cnecSensitivityProvider,
                                                                SensitivityAnalysisParameters sensitivityComputationParameters) {
-        LOGGER.debug("Sensitivity computation [start]");
-        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, workingStateId, cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
-        LOGGER.debug("Sensitivity computation [end]");
+        LOGGER.debug("Sensitivity analysis [start]");
+        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, network.getVariantManager().getWorkingVariantId(), cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
```
Comment (TheMaskedTurtle) [inline sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityAdapter.java:25]: This method is already defined in PowSyBl API you can remove the use of working variant ID

Comment (murgeyseb) [reply sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityAdapter.java]: done

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**